tests/components/spreadsheet.test.ts

```diff
@@ -395,3 +395,14 @@ describe("Composer / selectionInput interactions", () => {
     expect(model.getters.getSelectedZones()).toEqual([toZone("D1")]);
   });
 });
+test("cell popovers to be closed on clicking outside grid", async () => {
+  ({ model, fixture } = await mountSpreadsheet());
+
+  setCellContent(model, "A1", "=SUM(");
+  await nextTick();
+  await hoverCell(model, "A1", 400);
```

Comment:
The test fails when called by itself, not with the whole suit of test. this helper requires that we explcitely use `jest.fakeTImers()`.
It didn't fail when running the whole test file. because we did not properly clear the fake timers in test `Z-indexes of the various spreadsheet compo`
Don't forget to run every test you add/modify in standalone. ( cc @somu-odoo )